lib/resources/submissions.js

```diff
@@ -46,6 +46,25 @@ const findMultipart = (files) => {
 };
 const forceAuthFailed = Problem.translate(Problem.user.insufficientRights, Problem.user.authenticationFailed);
 
+// The following errors bubble up through multer from busboy, and are generic Error objects:
+const busboyErrors = [
+  'Malformed part header',
+  'Malformed content type',
+  'Malformed part header',
+  'Malformed urlencoded form',
+  'Missing Content-Type',
+  'Multipart: Boundary not found',
+  'Unexpected end of file',
+  'Unexpected end of form',
+];
+const multipartErrorHandler = (err, req, res, next) => {
+  if (busboyErrors.includes(err.message) || err.message.startsWith('Unsupported content type: ')) {
+    next(Problem.user.multipartParsingFailed());
```

Comment:
How about passing `err.message` to `Problem.user.multipartParsingFailed()` so that it can be included in the `details` object of the Problem response? If we have this extra information about what exactly went wrong, I figure we might as well return it to the user.